maker/src/routes.rs

```diff
@@ -242,6 +245,23 @@ pub async fn sync_on_chain(State(state): State<Arc<AppState>>) -> Result<(), App
     Ok(())
 }
 
+#[derive(Serialize)]
+pub struct Position {
+    tentenone: HashMap<ContractSymbol, Decimal>,
```

Comment:
What do you think of making this a custom struct with 2 fields instead of map? That would allow us to extend it easily, e.g. 

```
pub struct Position {
    positions: Vec<TenTenOnePosition>
}

pub struct TenTenOnePosition{
  symbol: String,
  quantity: Decimal
}

```

Reply:
Sure, can make it a HashSet instead, with the custom struct as the item, like you suggest.

Reply:
I guess the downside of that that the type system will not guarantee that we won't be  having multiple positions with the same ContractSymbel

Reply:
You can get the same behaviour by using a `HashSet` and implementing `Hash`, `PartialEq` and `Eq` on the custom struct only using the `ContractSymbol`. Obviously that's not as foolproof as deriving those traits on the key of a `HashMap`, but it's close enough!

Reply:
@bonomat: I did what you suggested, but the diff sucks because I force-pushed after rebasing as well. Sorry!

Reply:
No worries, the changeset is relatively small :)